backend/app/api/game.py

```diff
@@ -1,14 +1,165 @@
 import dataclasses
 import json
+import random
 from typing import Dict, List
 
-from fastapi import WebSocket, WebSocketDisconnect, APIRouter
-
 from app import schemas
-
+from fastapi import APIRouter, WebSocket, WebSocketDisconnect
 
 router = APIRouter()
 
+"""
+file holds all of the routes for the game behaviour
+"""
+
+points: Dict[str, int] = {
+    "A": 1, "B": 4, "C": 1, "D": 2, "E": 1, "F": 3, "G": 2, "H": 3, "I": 1, "J": 8, "K": 5, "L": 1, "M": 3, "N": 1,
+    "O": 1, "P": 4, "Q": 10, "R": 1, "S": 1, "T": 1, "U": 2, "V": 5, "W": 5, "X": 10, "Y": 4, "Z": 8
+}
+
+# 12 E's, 10 A's, 8 R's , 8 I's, 7 N's, 6 L's, 6 O's, 5 T's, 5 S's, 3 C's, 3 U's, 3 G's, 3 D's, 3 H's, 2 F's, 2 M's, 2 Y's, 2 P's,
+# 2 B's, 2 V's, 1 k, 1 W, 1 Z, 1 J, 1 X, 1 Q
+tiles = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "A", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "R", "R", "R", "R", "R",
+         "R", "R", "R", "I", "I", "I", "I", "I", "I", "I", "I", "N", "N", "N", "N", "N", "N", "N", "L", "L", "L", "L", "L", "L", "O", "O", "O",
+         "O", "O", "O", "T", "T", "T", "T", "T", "S", "S", "S", "S", "S", "C", "C", "C", "U", "U", "U", "G", "G", "G", "D", "D", "D", "H", "H",
+         "H", "F", "F", "M", "M", "Y", "Y", "P", "P", "B", "B", "V", "V", "K", "W", "Z", "J", "X", "Q"]
+
+# For keeping track of the number of turns
+num_turn: int = 2
+
+# If the search word returns a valid word, find its points and add it to the total
+# Otherwise return an exception
+def validate_Word():
+    word = find_Word(array1, array2)
```

Comment:
array1 and array2 are the arrays I used for comparing the boards when you're searching for a new word. I still have the Arrays I used for testing the code, should I add them here? Or alternatively, the validate_word() function could receive the 2 arrays as arguments, and then it could call the find_word() function using those arguments. Lmk whats the better option

Reply:
we definitely need them passed in as arguments, because the arrays are going to come from the frontend.